src/seb/cli/run.py

```diff
@@ -71,8 +73,8 @@ def build_model(model_name: str) -> seb.EmbeddingModel:
     logging_level=Arg("--logging-level", help="Logging level for the benchmark."),
 )
 def run_benchmark_cli(
-    model_name: str,
-    output_path: Path,
+    model_name: Optional[str] = None,
```

Comment:
Shouldn't we then add a flag for model names and allow it to be multiple. In that way it is equivalent to the tasks and languages.

Reply:
hmm right but this gives us a problem with the output path